bridges/AO3Bridge.php

```diff
@@ -61,9 +68,21 @@ public function collectData()
      */
     private function collectList($url)
     {
-        $html = getSimpleHTMLDOM($url);
+        $httpClient = RssBridge::getHttpClient();
```

Comment:
i'd prefer not using the http client directly but using `getSimpleHTMLDOM` or `getContents`.

Reply:
This was done prior to my commit to iirc respect AO3's scraping rules with regards to user agent. I just followed that and made it consistent.

Reply:
yes sorry i know